tools/src/launcher/Main.hs

```diff
@@ -96,6 +105,10 @@ data LauncherOptions = LO
 -- | The concrete monad where everything happens
 type M a = (HasConfigurations, HasCompileInfo) => Log.LoggerNameBox IO a
 
+data LType = LInfo | LNotice | LWarning | LError
+data NodeType = NWallet | NNode
```

Comment:
`NodeType` name looks confusing to me. This definition makes me think that `NWallet` is `cardano-node` (from `cardano-sl-wallet`) and `NNode` is `cardano-sl-node-simple` (from `cardano-sl-node`).
But `NWallet` is not a node at all, it's frontend!
Maybe call it `data Executable = EWallet | ENode`? Dunno, maybe there is better name. But wallet is not a node.

Reply:
Done